build.gradle

```diff
@@ -69,9 +70,13 @@ allprojects {
             }
         }
 
-        maven {
-            url = 'https://mvnrepository.com/artifact/com.icegreen/greenmail'
-        }
+//        maven {
+//            url = 'https://mvnrepository.com/artifact/com.icegreen/greenmail'
+//        }
+
+//        maven {
+//            url = 'https://mvnrepository.com/artifact/com.sun.activation/javax.activation'
+//        }
```

Comment:
Shall we remove the commented urls? Check other places as well.

Reply:
Removed.